components/brave_wallet_ui/components/desktop/account-filter-selector/account-filter-selector.tsx

```diff
@@ -71,12 +71,17 @@ export const AccountFilterSelector = ({
       onSelectAccount(account)
       return
     }
-    window.localStorage.setItem(LOCAL_STORAGE_KEYS.PORTFOLIO_ACCOUNT_FILTER_OPTION, account.id)
-    dispatch(WalletActions.setSelectedAccountFilterItem(account.id))
+    window.localStorage.setItem(LOCAL_STORAGE_KEYS.PORTFOLIO_ACCOUNT_FILTER_OPTION, account.accountId.uniqueKey)
+    dispatch(WalletActions.setSelectedAccountFilterItem(account.accountId.uniqueKey))
```

Comment:
Format is changed, is it ok to not to mess with migration?

Reply:
Can you make sure that an invalid selected account filter has a fallback to the "all" filter?